### DISCUSSION TITLE
Question about Github Student Developer Pack.

### DISCUSSION THREAD

**Discussion by ghost** [community/community#101951] [GitHub Education]
Question has been solved. It turns out that you just need to wait more time after your application has been approved so that you can access to the student benefits. Just be patient.

**Comment by sayidrajavi** [comment]
[logs_37258.zip](https://github.com/community/community/files/14049544/logs_37258.zip)
[TP-Link-Archer-C6U-main.zip](https://github.com/community/community/files/14049550/TP-Link-Archer-C6U-main.zip)
`[](url)`

**Reply by ghost** [reply]
What's this?
